pypopminer/CMakeLists.txt

```diff
@@ -12,19 +12,21 @@ if(NOT ${Boost_FOUND})
 endif()
 find_package(PythonInterp 3)
 find_package(PythonLibs 3)
-add_custom_target(pypopminer COMMENT "Building pypopminer")
+add_custom_target(src COMMENT "Building pyPopMiner")
```

Comment:
Why is target named src?

Reply:
IDE got carried away.. fixes incoming :)